src/main/java/rs/hybridit/App.java

```diff
@@ -3,9 +3,10 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @SpringBootApplication
-
+@EnableSwagger2
```

Comment:
best practice is to register separate configuration class

https://www.baeldung.com/swagger-2-documentation-for-spring-rest-api